### ISSUE TITLE
[Bug] Xamarin.Forms TabIndex not working on Android and iOS

### ISSUE THREAD

**Issue by davantigroup** [xamarin/Xamarin.Forms#7568]
### Sample

https://github.com/xamarin/xamarin-forms-samples/tree/master/UserInterface/Accessibility

### Description

It seems the TabIndex is not working anymore, even in the sample provided by Microsoft regarding to Keyboard Accessibility it only focuses the first button, it works properly for entry or stepper tough.

### Steps to Reproduce

1. Open the sample project
2. Go to Tab Order
3. Press Tab multiple times
4. When the focus is on the first button press Tab again

### Expected Behavior

The focus should move to the next button

### Actual Behavior

The focus is stuck on the first button

### Information
  - Platform with the issue: 
  - iOS:  12.4
  - Android:  8.1, 9.0
![image](https://user-images.githubusercontent.com/54096887/65041058-88fdd300-d95e-11e9-8f8f-6178a0864db9.png)

**Comment by rnic99** [comment]
@samhouts Id like to add some extra to this bug, in the same accessibility app we have two different pages, "Automation Properties" and "Tab Order".  The bug was originally reported for TabIndex not working on the sample app. Testing the accessibility app in XF 4.3 I was not having any issues navigating on the Tab Order sample but I have encountered an interesting bug in the accessibility demo portion of the app, and its very similar to the one described here. When trying to navigate using a screen reader I have found that I cannot navigate past the "Enter your name:" label when its wrapped in a stack layout. However when I removed the stack layout i could freely navigate past this control. Ive found that wrapping other controls in stack layouts either perform identical or similar issues in regards to how the screen reader behaves with with the ui elements. 
However I have found that wrapping multiple elements in stack layouts at once also produces different results.

Not entirely sure if this warrants another bug or similar enough to this one. I can easily create another bug if need be.

**Comment by samhouts** [comment]
@rnic99 Yeah, I would open a new bug for that. Thanks!!!

**Comment by rnic99** [comment]
@samhouts [https://github.com/xamarin/Xamarin.Forms/issues/8613](url)

**Comment by samhouts** [comment]
@davantigroup It looks like TabIndex behaves the same on iOS no matter what version I'm using. Are you really seeing this issue on iOS, too? Thanks!

**Comment by davantigroup** [comment]
@samhouts  We had the same issue on iOS too, we can test it again if needed

**Comment by samhouts** [comment]
Please do! If you can post a gif or video, too, that would be immensely helpful! Thanks!!

**Comment by samhouts** [comment]
@davantigroup Also, if you could test with the package attached to https://github.com/xamarin/Xamarin.Forms/pull/9702#issuecomment-593727682 and let me know if it improves or worsens the situation, that would be great. Thanks!

**Comment by E75** [comment]
@samhouts It doesn't work also in UWP app

**Comment by Suplanus** [comment]
Any updates? I think Accessibility should be prio high.
What thinks @magoolation about this?

**Comment by rachelkang** [comment]
The following is a summary of the current behavior I am observing on XF 5.0.0.1874 for TabIndex, as per the "Tab order" page in this sample (with default settings / no buttons activated):
| Platform | Focus order | Keyboard tabbing | Misc |
|---------|--------------|-------------------|------|
| iOS | Right swiping to navigate the focus with VO seems to work properly. The focus order goes through the elements as expected, left to right, top to bottom. | TabIndex behavior is strange. Focus starts on "Descending TabIndex" button, then goes in the following order: Date picker (which is opened), "Enter data" entry, "Enter name" entry, Monkey picker (which is opened), SearchBar [text cursor is in search field, but focus is still on monkey picker], Time picker (which is opened), and then back to Date picker, etc. // Removing all the TabIndex code from the XAML does nothing - the behavior remains the same. | |
| Android | Exactly the same as above. | Focus remains stuck on the first button, as indicated in the original issue. // Removing all the TabIndex code from the XAML slightly improves this - the focus moves through all the buttons and reaches the date picker. Once the date picker is reached, the date picker is opened; when it is closed, the focus either goes to the top of the page (on clicking "Cancel") or remains on the same element (on clicking "OK").| Focus from keyboard and focus from tapping screen seem to be completely separate (?) If I navigate focus to some element A with keyboard tab, and then tap screen to focus on some element Z, when I use the keyboard tab button again, the focus jumps to element B (element after A rather than element after Z as would be expected)  |
| UWP | Exactly the same as above. | Same as left. However, doesn't stop at last element and cycles through.  | Sometimes, focus is seen on an invisible ghost element below the TimePicker, or on the entire page. Can't navigate to Window/AppBar solely with keyboard tabbing alone  |

We will continue to explore this issue from here. If anyone has different or additional observations, please do share!
Thank you everyone for your continued patience as we try to better understand and investigate the best solution for the TabIndex / focus order paradigm.

**Comment by PureWeen** [comment]
For this issue please try

https://github.com/xamarin/XamarinCommunityToolkit/issues/1087
